modules/search-cli/src/main/scala/io/renku/search/cli/perftests/GitLabEntities.scala

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright 2024 Swiss Data Science Center (SDSC)
+ * A partnership between École Polytechnique Fédérale de Lausanne (EPFL) and
+ * Eidgenössische Technische Hochschule Zürich (ETHZ).
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.renku.search.cli.perftests
+
+import cats.syntax.all.*
+import io.bullet.borer.Decoder
+import io.bullet.borer.Decoder.DecodingError
+import io.bullet.borer.NullOptions.given
+import io.bullet.borer.derivation.MapBasedCodecs.deriveDecoder
+import io.renku.search.model.projects
+
+import java.time.Instant
+import scala.util.Try
+
+final private case class GitLabProject(
+    id: Int,
+    description: Option[String] = None,
+    name: String,
+    path_with_namespace: String,
```

Comment:
If you like to use camel-case, there is a `@key` annotation in `borer.derivation` where you could specify the json property name (and use whatever name in the case class)

Reply:
I see, I remembered you told me it's possible but couldn't find it in the docs easily. In the end, I thought, it's so internal that I probably don't care too much. Actually, it's having this one-to-one shape on the GL response is not too bad.